### PR TITLE
Improve stream presentation

### PR DESCRIPTION
**Pull request by dbernard**
Minor improvements to stream presentation:

 - Sort live streams by viewer count
 - Display message (rather than blank page) when no streamers are online

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -97,7 +97,8 @@ def get_streams(streamers):
         except requests.exceptions.HTTPError as htp:
             app.logger.error("Couldn't load user '%s': %s", user, htp)
 
-    return filter(lambda s: s['playing'] != 'Offline', streams)
+    streams.sort(key=lambda s: s['viewers'], reverse=True)
+    return list(filter(lambda s: s['playing'] != 'Offline', streams))
```
Comment (arusahni) [inline app.py]: These two lines can be combined:

`list(sorted((s for s in streams if s['playing'] != 'Offline'), key=lambda s: s['viewers'], reverse=True))`

**Comment by arusahni** [comment]
👍 looks good - just the nitpick about sorting.

**Comment by dbernard** [comment]
@arusahni is there a comment somewhere that I'm not seeing for that sorting nitpick?

**Comment by arusahni** [comment]
@dbernard yes - I can see it on the desktop site. If you're mobile, maybe it's not visible?

**Comment by arusahni** [comment]
@dbernard ahh. I think the "review" mode in GitHub required me to submit a set of comments even though I had only made one. It should be visible, now.

**Comment by dbernard** [comment]
Code updated to use that juicy one-liner.

**Comment by arusahni** [comment]
Looks good!

**Pull request merged**